README.md

```diff
@@ -43,12 +152,221 @@ docker build .
+  Downloading typing_extensions-4.4.0-py3-none-any.whl (26 kB)
+Requirement already satisfied: MarkupSafe>=2.0 in /usr/local/lib/python3.9/site-packages (from Jinja2>=2.10.3->datasette->-r ./requirements.txt (line 1)) (2.1.1)
+Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.9/site-packages (from python-dateutil->sqlite-utils->-r ./requirements.txt (line 2)) (1.16.0)
+Collecting h11>=0.8
+  Downloading h11-0.14.0-py3-none-any.whl (58 kB)
+     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 58.3/58.3 KB 1.1 MB/s eta 0:00:00
+Collecting anyio<5.0,>=3.0
+  Downloading anyio-3.6.2-py3-none-any.whl (80 kB)
+     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 80.6/80.6 KB 1.5 MB/s eta 0:00:00
+Building wheels for collected packages: python-multipart
+  Building wheel for python-multipart (setup.py): started
+  Building wheel for python-multipart (setup.py): finished with status 'done'
+  Created wheel for python-multipart: filename=python_multipart-0.0.5-py3-none-any.whl size=31678 sha256=897c5a5dd92a40959d33afdd07743e367462af1eb5bbb386da24905fab8d71d6
+  Stored in directory: /root/.cache/pip/wheels/fe/04/d1/a10661cc45f03c3cecda50deb2d2c22f57b4e84a75b2a5987e
+Successfully built python-multipart
+Installing collected packages: sqlite-fts4, rfc3986, pytz, urllib3, typing-extensions, tabulate, sniffio, PyYAML, python-multipart, python-dateutil, pluggy, pint, numpy, mergedeep, idna, hupper, h11, et-xmlfile, click-default-group-wheel, charset-normalizer, certifi, asgiref, aiofiles, uvicorn, sqlite-utils, requests, pandas, openpyxl, janus, asgi-csrf, anyio, httpcore, httpx, datasette
+Successfully installed PyYAML-6.0 aiofiles-22.1.0 anyio-3.6.2 asgi-csrf-0.9 asgiref-3.5.2 certifi-2022.12.7 charset-normalizer-2.1.1 click-default-group-wheel-1.2.2 datasette-0.63.2 et-xmlfile-1.1.0 h11-0.14.0 httpcore-0.16.2 httpx-0.23.1 hupper-1.10.3 idna-3.4 janus-1.0.0 mergedeep-1.3.4 numpy-1.23.5 openpyxl-3.0.10 pandas-1.5.2 pint-0.20.1 pluggy-1.0.0 python-dateutil-2.8.2 python-multipart-0.0.5 pytz-2022.6 requests-2.28.1 rfc3986-1.5.0 sniffio-1.3.0 sqlite-fts4-1.0.3 sqlite-utils-3.30 tabulate-0.9.0 typing-extensions-4.4.0 urllib3-1.26.13 uvicorn-0.20.0
+    WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
+    WARNING: You are using pip version 22.0.4; however, version 22.3.1 is available.
+    You should consider upgrading via the '/usr/local/bin/python -m pip install --upgrade pip' command.
+Removing intermediate container 9bf12e2e0869
+ ---> 6b14dab7e5d3
+Step 14/14 : CMD ["python3", "flask_main.py"]
+ ---> Running in c0fcb81b43c2
+Removing intermediate container c0fcb81b43c2
+ ---> 3b2061d454a2
+Successfully built 3b2061d454a2
+
+Use 'docker scan' to run Snyk tests against images to find vulnerabilities and learn how to fix them
+```
+</details>
+
+## Example runthrough
```

Comment:
We don't need an example runthrough, please remove this section

Reply:
Hidden by default. Totally extra